tests/cloner_test.go

```diff
@@ -750,6 +751,104 @@ var _ = Describe("Namespace with quota", func() {
 	})
 })
 
+var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:component]Cloner Test Suite", func() {
+	f := framework.NewFrameworkOrDie(namespacePrefix)
+
+	var sourcePvc *v1.PersistentVolumeClaim
+	var targetPvc *v1.PersistentVolumeClaim
+
+	AfterEach(func() {
+		if sourcePvc != nil {
+			By("[AfterEach] Clean up source PVC")
+			err := f.DeletePVC(sourcePvc)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		if targetPvc != nil {
+			By("[AfterEach] Clean up target PVC")
+			err := f.DeletePVC(targetPvc)
+			Expect(err).ToNot(HaveOccurred())
+		}
+	})
+
+	It("[test_id:3999] Create a data volume importing an image and then clone it and verify retry count", func() {
```

Comment:
Technically we are not importing an image first here, we are filling it with some data using the filler pod. Also where did the test_id come from?

Reply:
right, 
removing the "importing" part from name, 
test_id comes from test plan